Add ROSCon2019 post. (#82)
micro-ROS/micro-ROS.github.io: _posts/2019-11-01-micro-ROS-at-ROSCon2019.md

---
title: micro-ROS at ROSCon 2019 Macau
author: borjaouterelo 
---

> ![ROSCon2019Macu](/img/posts/ROSConMacau.png){:.img-responsive and style="max-width: 50%; margin-left: auto; margin-right: auto;"}

Reviewer: ```suggestion
![ROSCon2019Macau](/img/posts/ROSConMacau.png){:.img-responsive and style="max-width: 50%; margin-left: auto; margin-right: auto;"}
```

ROSCon ([https://roscon.ros.org](https://roscon.ros.org/)) is the most important event of the ROS community. 
The 2019 edition was held in Macau, China.

> This edition has been the biggest to date with more than 500 attendants.

Reviewer: ```suggestion
This edition has been the biggest to date with about 700 attendants.
```
Karsten Knese mentioned about 700 participants.

Author: Thanks, I was looking for the data but haven't found official numbers. Also interesting will be the streaming viewers.

> During this year's event, micro-ROS was present in two different ways a talk and booths on the exhibition floor.

Reviewer: ```suggestion
During this year's event, micro-ROS was present in two different ways: a talk and booths on the exhibition floor.
```

## Talk

![Ingo&BorjaPresentation](https://pbs.twimg.com/media/EIRm9RPXYAAKwnN?format=jpg&name=large){:.img-responsive and style="max-width: 60%; float: right; padding-left: 10px"}
During the second day, Ingo and Borja presented micro-ROS to the ROS community.

> They presented the current status of the framework and supplement with some rationale on why to use it and why we need microcontrollers in Robots.

Reviewer: ```suggestion
They presented the current status of the framework and supplement with some rationale on why to use it and why we need microcontrollers in robots.
```

## Exhibition

> Within the Exhibition floor, there were two booths with material directly related to micro-ROS, Bosch Corporate Research booth, and eProsima's.

Reviewer: ```suggestion
Within the exhibition floor, there were two booths with material directly related to micro-ROS, Bosch Corporate Research booth, and eProsima's.
```

> In the Bosch booth, they had the next evolution of the Community demo, Kobuki, with different hardware setup and some premier of the upcoming updates to come.

Reviewer: ```suggestion
In the Bosch booth, they had the next evolution of the micro-ROS community demo based on a Kobuki platform with different hardware setup and some premier of the upcoming updates to come.
```

The eProsima booth was divided into two different spaces, one dedicated to Fast RTPS, their well known DDS implementation powering ROS 2 communications.
The other space was dedicated to microcontrollers development with Micro XRCE-DDS and micro-ROS as main attractions.
In this "micro" area, there was a TV showing a reel presentation of micro-ROS, the video prepared by FIWARE marketing team, and PIAP's video on benchmarking.
Also, in this area, and grabbing almost all the attraction of the public, there were two different micro-ROS demonstrations:
* A first demonstration using a Crazyflie 2.1 and sending data to the RVIz, which was visualized on the TV screen.

> * A second Crazyflie 2.1 issuing commands to a TurtleBot 2 platform to move it.

Reviewer: ```suggestion
* A second Crazyflie 2.1 issuing commands to a Kobuki platform to move it.
```
